src/sparseml/pytorch/sparsification/distillation/modifier_per_layer.py

```diff
@@ -155,6 +155,8 @@ def __init__(
         self._teacher_output_shapes: Dict[str, torch.Size] = {}
         self._loaded_projection = None
 
+        self._compare_model_output = False
```

Comment:
Could we set up a way to override this? If my understanding is correct this would not check the types for  all usages of `PerLayerDistillationModifier`? Does my argument make sense?

Reply:
It only skips the type check for the final model output, not for the layers in between. This final output actually goes unused by `PerLayerDistillationModifier`